includes/hub/admin/class-nodes-list.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * Newspack Hub Event Log List Table
+ *
+ * @package Newspack
+ */
+
+namespace Newspack_Network\Hub\Admin;
+
+use Newspack_Network\Hub\Nodes;
+use Newspack_Network\Hub\Node;
+
+/**
+ * The Event Log List Table
+ */
+class Nodes_List {
+
+	/**
+	 * Initialize this class and register hooks
+	 *
+	 * @return void
+	 */
+	public static function init() {
+		add_filter( 'manage_' . Nodes::POST_TYPE_SLUG . '_posts_columns', [ __CLASS__, 'posts_columns' ] );
+		add_action( 'manage_' . Nodes::POST_TYPE_SLUG . '_posts_custom_column', [ __CLASS__, 'posts_columns_values' ], 10, 2 );
+		add_action( 'admin_enqueue_scripts', [ __CLASS__, 'admin_enqueue_scripts' ] );
+		add_action( 'admin_bar_menu', [ __CLASS__, 'admin_bar_menu' ], 100 );
+	}
+
+	/**
+	 * Modify columns on post type table
+	 *
+	 * @param array $columns Registered columns.
+	 * @return array
+	 */
+	public static function posts_columns( $columns ) {
+		unset( $columns['date'] );
+		unset( $columns['stats'] );
+		$columns['links'] = __( 'Useful links', 'newspack-network' );
```

Comment:
Very nitpicky, but I'm not sold on "Useful links" as a name for this column. Perhaps just "Links" or something else like "Manage"